build/integration.sh

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if [[ ! -z "${JENKINS_HOME-}" ]]
+then 
+    exec ./jenkins_e2e.sh
```

Comment:
The exit code is the reason for exec. I can catch the exit code and return that too, if exec has issues.